ARCHITECTURE.md

```diff
@@ -0,0 +1,113 @@
+# DSO Toolkit architecture
+
+Dit document beschrijft de visie en architectuur van het DSO Toolkit landschap.
+
+Het begint met een samenvatting waarna per onderdeel een uitgebreide beschrijving volgt.
+
+## Synopsis
+
+De DSO Toolkit is een Design System en bestaat uit de volgende NPM packages:
+
+* `@dso-toolkit/sources`
+* `dso-toolkit`
+* `@dso-toolkit/css`
+* `@dso-toolkit/core`
+* `@dso-toolkit/react`
+* `@dso-toolkit/leaflet`
+* `@dso-toolkit/react-leaflet`
+
+![DSO Toolkit dependency graph](dependency-graph.svg)
+
+## Termen
+
+* **implementatie**: Binnen de context van de toolkit bedoelen we hier een *type uitvoering*: Een component heeft één of meerdere implementaties: HTML/CSS component, Web Component, React Component, Angular Component. Buiten de context van de toolkit kan een afnemer van de toolkit een component binnen zijn applicatie implementeren.
+* **maintainer**: Een maintainer beheert de codebase van de DSO Toolkit.
+* **component**: De DSO Toolkit maakt geen gebruik van een methode zoals Atomic Design: Binnen de toolkit is alles een component. Van een heading zoals `h1` tot een complexe werkvorm zoals Accordion.
+
+## Sources
+
+Deze package is de basis voor de andere packages in de DSO Toolkit suite. Hier centralizeren we de functionaliteit die door maintainers wordt gebruikt:
```

Comment:
Hier hanteer je de term suite, elders 'landschap' en eigenlijk is wat we hebben ook een 'systeem' (namelijk 'Design System'). Is die woordkeuze overal bewust?

Reply:
centralizeren -> centraliseren

Reply:
"Landschap" heb ik aangepast naar "Design System", veel beter.

Met _suite_ bedoel ik alle verschillende packages: De MS Office Suite bestaat ook uit Word, Excel, etc. Maar "Design System" leest net zo goed, dus ik heb 'm ook aangepast.

Reply:
Prima!